model/jpa/src/main/java/org/keycloak/models/jpa/RealmAdapter.java

```diff
@@ -35,6 +35,7 @@
 import jakarta.persistence.EntityManager;
 import jakarta.persistence.LockModeType;
 import jakarta.persistence.TypedQuery;
+
 import java.util.*;
```

Comment:
Should this be fixed as well? :)

Reply:
If we take this apart, there will be more than five.  
I think it is more reasonable to use * directly in this case.

Reply:
We have an unwritten rule not to use star imports anywhere in the project. :)

Reply:
Thanks